ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,29 @@
+## Bug Report
+
+### Description
+A clear and concise description of what the bug is.
+
+### Steps to reproduce
+
+1. How to setup a proper environment to reproduce the bug, preferably, the explicit commands to setup said environment
+2. Ensure correct environmental configuations, such as files, permissions, licenses, development variables, etc
+3. Start "X" required service(s) required to reproduce the bug
+4. Run the following commands and/or steps to reproduce the bug
+...
+
+### Expected result
+A clear and concise description of what you expected to happen.
+
+### Actual result
+What is the actual behaviour you saw? Are there logs?
+
+### Possible solution
+<!--- Only if you have valid suggestions on a fix for the bug and have thorough knowledge of the code base -->
+
+### Additional information
+For example, screenshots or analysis so far.
+
+### Environment
+OS:
+Browser:
+Stack used and version:
```

Comment:
I think used H/W is missing on this list...

Reply:
BTW missing versioning/tagging of docker containers in such cases...

Reply:
Add an impact - Level 1,2 or 3 where:
1- Stopping BuG - need immediate action (daily)
2- Important - to be handled ASAP (week or two)
3- Minor - Can be handled (release or later)